### ISSUE TITLE
Incorrect Adding of Messages to the Message Stack in Miranda

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#976]
**Reported by parham on 2010-10-11 19:16**
When chatting in Miranda and using NVDA, it has happened many times for me that when pressing ctrl+insert+1/2/3, instead of just one message, two messages are read. I have tried for a long time to find out when this happens, but I have not been successful to pinpoint the exact times this occurs. The most I have noticed about this is when two messages arrive and get sent very quickly (E.G. in less than a second).

**Comment by ehollig** [comment]
Miranda has been superseded by Miranda NG. If this still occurs with the latest Miranda, please provide a log or open a new ticket.

**Comment by leonardder** [comment]
I guess we can leave this open for Miranda NG as well. Nevertheless, this issue is very old and abandoned.

Anyone with Miranda who can reproduce this? I'll leave this open for next week and will close if we haven't gotten any similar reports. This issue can always be reopened if this is still the case.

Also note that this probably depends on what messaging plugin one is using. For example, Scriver can be set up to print the name of the sending user only once, thus not for every message.

**Comment by PratikP1** [comment]
I have been using MirandaNG for at least a year. I have never encountered this issue.

**Comment by derekriemer** [comment]
adds wait for a close

**Comment by jcsteh** [comment]
Closing as worksforme as per https://github.com/nvaccess/nvda/issues/976#issuecomment-315557184. If the problem persists, please comment here and we can try to provide steps to debug further.
